tests/workers_round_robin_bench.ts

```diff
@@ -0,0 +1,72 @@
+const data = "HTTP/1.1 200 OK\r\nContent-Length: 12\r\n\r\nHello World\n";
```

Comment:
Can you give some summary comment at the top of this file explaining what this measures?